packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-phone/java/integrationsteps.md

```diff
@@ -87,9 +89,43 @@ You need to build a form to capture the user's SMS verification code. For exampl
 
 </div>
 
-### Step 5: User enters SMS code
+### Step 4 (voice feature alternative): User enters phone number and phone factor method
+
+This step assumes that your org is enabled with the voice feature.
+
+You need to build a form to capture the user's phone number as well as a subsequent form for the user to select their phone verification method (either SMS or voice).
+
+<div class="common-image-format">
+
+![Displays the Java SDK's enroll phone number authenticator form](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-self-serv-screen-verify-phone-num-java.png)
+
+</div>
+
+> **Note:** The Java SDK requires the following phone number format: `{+}{country-code}{area-code}{number}`. For example, `+15556667777`.
+
+<div class="common-image-format">
+
+![Displays the Java SDK's phone factor (SMS or voice) form](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-self-serv-screen-verify-phone-mode-java.png)
+
+</div>
+
+When the user enters their phone number and selects SMS to receive the verification code, capture this information and send it to the [`IDXAuthenticationWrapper.submitPhoneAuthenticator()`](https://github.com/okta/okta-idx-java/blob/master/api/src/main/java/com/okta/idx/sdk/api/client/IDXAuthenticationWrapper.java#L398) method. For example:
```

Comment:
There's actually 2 ways to do this. We do it _slightly_ differently in the samples, but I think this is far simpler to explain.

Reply:
So it this ok for now? Let me know if this needs to change or if there is an addition method that we need to explain.

Reply:
I'd leave it this way 👍